tests/configs/lua-block-simple/nginx.conf

```
@@ -31,7 +31,7 @@ http {
     upstream foo {
         server 127.0.0.1;
         balancer_by_lua_block {
-            -- use Lua to do something interesting here
+            -- use Lua that'll do something interesting here
```

```suggestion
            -- use Lua that'll do something interesting here {
```
It might also be nice to add a single `{` to a comment or two in the fixtures to make sure that also doesn't break stuff